<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_add_palette"/>
    <addaction name="action_save"/>
    <addaction name="action_load"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_load">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_add_palette">
   <property name="text">
    <string>Добавить палитру</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
